src/main/java/com/esri/samples/geometry/convex_hull_list/README.md

```diff
@@ -0,0 +1,37 @@
+<h1>Convex Hull List</h1>
+
+<p>Generate convex hull polygon(s) from multiple input geometries. As a visual analogy, consider a set of points as nails in a board. The convex hull of the points would be like a rubber band stretched around the outermost nails.</p>
```

Comment:
The visual analogy part can go in a separate `p` tag. The first p tag should always be pretty short for the sample viewer short description.